lib/planet.js

```diff
@@ -0,0 +1,60 @@
+const data = require('../out/data.json')
+const feedFinder = require('find-rss')
+const fs = require('fs')
+const feedParser = require('feed-read-parser')
+const gciStart = new Date('November 27, 2017')
+
+const errs = []
+let articles = []
```

Comment:
This isn't used outside of the function right?

Reply:
There's a better way of doing recursion than this. But it's ok for now, no changes.